client/app/assets/less/inc/base.less

```diff
@@ -23,10 +23,18 @@ body {
     background: #F6F8F9;
     font-family: @redash-font;
     position: relative;
-    
+
+    app-view {
+      padding-bottom: 15px;
+    }
+
     &.headless {
         padding-top: 10px;
-        
+
+        app-view {
+          padding-bottom: 0;
```

Comment:
Why is it needed?

Reply:
Previously in headless mode padding was removed as well - I just restored that behavior

Reply:
Perhaps it has to do with the iframe version not having a needless vertical scroll or sth